pandas/core/groupby.py

```diff
@@ -1504,7 +1504,11 @@ def aggregate(self, values, how, axis=0):
                 raise NotImplementedError
             out_shape = (self.ngroups,) + values.shape[1:]
 
-        if is_numeric_dtype(values.dtype):
+        if is_datetime:
```

Comment:
yep, I think that should be cause before here though (iow there is a test for it now that asserts that it fails)